### PR TITLE
feature: prepare production achievements

### PR DESCRIPTION
**Pull request by dhenkel92**
The idea is that we can copy & execute the `graphql-seed` & `graphql-enable` mutations in our production apollo playground.

The alternative would be writing a script to execute the queries, but I don't see an advantage.

### CODE REVIEW

File: common/achievement/template.ts [hunk 1]
```diff
@@ -168,11 +168,11 @@ export async function checkTemplateConsistencyBeforeActivating(template: Achieve
     logger.info(`Checking AchievementTemplate(${template.id}) for consistency`, { template, group });
 
     // ---- Template Metadata -----
-    if (!template.title || !template.description || !template.subtitle) {
+    if (!template.title || !template.description) {
```
Comment (dhenkel92) [inline common/achievement/template.ts:171]: Subtitle is not required

File: common/achievement/template.ts [hunk 2]
```diff
@@ -168,11 +168,11 @@ export async function checkTemplateConsistencyBeforeActivating(template: Achieve
...
         throw new PrerequisiteError(`AchievementTemplates need a name, description and subtitle`);
     }
 
-    const actionFields = [!!template.actionName, !!template.actionRedirectLink, !!template.actionType];
+    const actionFields = [!!template.actionName, !!template.actionType];
```
Comment (dhenkel92) [inline common/achievement/template.ts:175]: There might be actions that don't have a link, like waiting for course approval.

File: common/achievement/template.ts [hunk 3]
```diff
@@ -192,7 +192,7 @@ export async function checkTemplateConsistencyBeforeActivating(template: Achieve
             throw new PrerequisiteError(`Aggregation ${name} uses unknown metric ${aggregation.metric}`);
         }
 
-        if ((template.type === AchievementType.TIERED) !== !!aggregation.valueToAchieve) {
+        if ((template.type === AchievementType.TIERED) !== (typeof aggregation.valueToAchieve === 'number')) {
```
Comment (dhenkel92) [inline common/achievement/template.ts:195]: This was failing if valueToAchieve is 0

File: graphql/achievement_template/mutations.ts [hunk 4]
```diff
@@ -39,8 +38,8 @@ class AchievementTemplateCreateInput implements AchievementTemplateCreate {
 
     @Field()
     condition: string;
-    @Field((type) => JSONResolver)
-    conditionDataAggregations: ConditionDataAggregations;
+    @Field()
+    conditionDataAggregations: string;
```
Comment (dhenkel92) [inline graphql/achievement_template/mutations.ts:42]: I wasn't able to generate the graphql json scalar via node. Therefore, I switched the input type to string and move the parsing to the resolver function.

**Review by Jonasdoubleyou — APPROVED** [review]

**Pull request merged**